crypto/src/main/scala/org/bitcoins/crypto/CryptoUtil.scala

```diff
@@ -14,18 +14,34 @@ import scodec.bits.{BitVector, ByteVector}
   */
 trait CryptoUtil {
 
+  def normalize(str: String): String = {
+    java.text.Normalizer.normalize(str, java.text.Normalizer.Form.NFC)
+  }
+
+  def serializeForHash(str: String): ByteVector = {
+    ByteVector(normalize(str).getBytes("UTF-8"))
+  }
```

Comment:
Should these be private?

Reply:
I'd prefer that they not be (in case I want to generate test vectors in the future for example, and also for debugging in the future)